src/main/kotlin/com/openlattice/users/Auth0Utils.kt

```diff
@@ -28,6 +28,7 @@ fun getUpdatedUsersPage(
                     .withSearchEngine(SEARCH_ENGINE_VERSION)
                     .withQuery("$UPDATED_AT={$lastSync TO $currentSync]")
                     .withFields("$USER_ID,$EMAIL,$NICKNAME,$APP_METADATA,$IDENTITIES", true)
+                    .withFields("$NAME,$GIVEN_NAME,$FAMILY_NAME", true)
```

Comment:
Is there a reason this isn't on the previous `withFields` line / does this for sure not overwrite that?

Reply:
Not a good reason, just to avoid long lines. I'm trying to test locally to make sure it doesn't override.

Reply:
Looking at the parent class it looks like this will override.

Reply:
Roger - fixing.